Header header
# /// Right measurement
uint16 Right
# /// Left measurement
uint16 Left
